GameOfLife.rb

```diff
@@ -1,99 +1,36 @@
-class GameOfLife
```

Comment:
Classname is correct, but filename should be named `game_of_life.rb` following Ruby conventions